SimulationConditions/inc/SimBookkeeper.hh

```diff
@@ -0,0 +1,59 @@
+#ifndef SimulationConditions_SimBookkeeper_hh
+#define SimulationConditions_SimBookkeeper_hh
+
+//
+// SimBookkeeper is the ProditionsEntry for
+//
+
+// Mu2e includes
+#include "Mu2eInterfaces/inc/ProditionsEntity.hh"
+#include "fhiclcpp/ParameterSet.h"
+
+namespace mu2e {
+
+  class SimBookkeeper : virtual public ProditionsEntity {
```

Comment:
I think using virtual here can mask a potential problem in the future.  I see some other ProditionsEntities use virtual and unless someone wants to defend it, I'll remove it there too.

Reply:
I've removed "virtual" from this file